repository_service_tuf_api/config.py

```diff
@@ -27,7 +27,7 @@
 example_update_settings = json.loads(content)
 
 
-class Response(BaseModel):
+class GetResponse(BaseModel):
```

Comment:
Can you move the response closer to the `get()` function, so when you check the `get()` function you can easily hover to see the `GetResponse`?
I wrote it in the last comment, but it seems you missed it :D